server/src/test/java/org/opensearch/cluster/coordination/ClusterFormationFailureHelperTests.java

```diff
@@ -818,8 +818,8 @@ public void testDescriptionAfterBootstrapping() {
             )
         );
 
-        final DiscoveryNode otherMasterNode = new DiscoveryNode("other-master", buildNewFakeTransportAddress(), Version.CURRENT);
-        final DiscoveryNode otherNonMasterNode = new DiscoveryNode(
+        final DiscoveryNode otherClusterManagerNode = new DiscoveryNode("other-master", buildNewFakeTransportAddress(), Version.CURRENT);
```

Comment:
`other-master` -> `other-cluster-manager`

Reply:
Thanks for catching it. I forgot it again in my last round of scan. 😂

Reply:
Changed the name. 👍